notification-server/src/main/java/uk/ac/bbsrc/tgac/miso/notification/dto/NotificationDto.java

```diff
@@ -0,0 +1,169 @@
+package uk.ac.bbsrc.tgac.miso.notification.dto;
+
+import java.nio.file.Path;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.As;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+
+import uk.ac.bbsrc.tgac.miso.core.data.type.HealthType;
+
+@JsonTypeInfo(use = Id.CLASS, include = As.PROPERTY, property = "class")
+public abstract class NotificationDto {
+
+  private String runName;
+  private Path sequencerFolderPath;
+  private String sequencerName;
+  private String containerId;
+  private int laneCount;
+  private HealthType healthType;
+  private DateTime startDate;
+  private DateTime completionDate;
+  private boolean pairedEndRun;
+  private String software;
+
+  public String getRunName() {
+    return runName;
+  }
+
+  public void setRunName(String runName) {
+    this.runName = runName;
+  }
+
+  public String getSequencerName() {
+    return sequencerName;
+  }
+
+  public void setSequencerName(String sequencerName) {
+    this.sequencerName = sequencerName;
+  }
+
+  public DateTime getCompletionDate() {
+    return completionDate;
+  }
+
+  public void setCompletionDate(DateTime completionDate) {
+    if (completionDate != null) {
+      this.completionDate = new DateTime(completionDate, DateTimeZone.UTC);
```

Comment:
Using `java.util.Date` would work better since that's what they would be converted to for use in MISO Runs anyway. The less conversions we need to do the better.

Hard-coding the timezone to UTC might help when the DB is running on UTC, but it could cause inconsistencies with a DB not running on UTC.